elements/pl-overlay/pl-overlay.py

```diff
@@ -0,0 +1,124 @@
+import prairielearn as pl
+import lxml.html
+import chevron
+
+
+VALIGN_DEFAULT = 'middle'
+HALIGN_DEFAULT = 'center'
+CLIP_DEFAULT = True
+
+VALIGN_VALUES = ['top', 'middle', 'center', 'bottom']
+HALIGN_VALUES = ['left', 'middle', 'center', 'right']
+
+# Percent to translate each alignment by.  This is relative to the top-left corner of the element.
+ALIGNMENT_TO_PERC = {
+    'top': '0%',
+    'left': '0%',
+    'middle': '-50%',
+    'center': '-50%',
+    'bottom': '-100%',
+    'right': '-100%'
+}
+
+
+def prepare(element_html, data):
+    element = lxml.html.fragment_fromstring(element_html)
+    #
```

Comment:
Should there be a comment here?

Reply:
I have no idea what I was trying to do here :shrug: